src/lib/Scenes/Artwork/Components/ArtworkDetails.tsx

```diff
@@ -19,22 +21,34 @@ export class ArtworkDetails extends React.Component<ArtworkDetailsProps> {
     context_module: Schema.ContextModules.ArtworkDetails,
   }))
   render() {
+    const { artwork } = this.props
+
+    const enableLotConditionReport =
+      NativeModules.Emission &&
+      NativeModules.Emission.options &&
+      NativeModules.Emission.options.AROptionsLotConditionReport
```

Comment:
we can use optional chaining here

```ts
const enableLotConditionReport = NativeModules.Emission?.options?.AROptionsLotConditionReport
```

Reply:
👍 I was using convention here: https://github.com/artsy/emission/blob/master/docs/developing_a_feature.md but I agree optional chaining is better. I'll update here and I want to follow up with some clean up for lab options in general including this, we have a bunch that we aren't using or are incorrectly defined in Eigen.